pkg/controllers/gacha-controller.go

```diff
@@ -5,63 +5,27 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"strconv"
 
-	"github.com/gorilla/mux"
 	"github.com/iamananya/ginco-task/pkg/models"
 	"github.com/iamananya/ginco-task/pkg/utils"
 )
 
 var NewUser models.User
-
-/*
-TODO----(Task completed)
-
-[*] Handle case for empty name- return error (Error code 400), non exisiting user ID return error
-[*] Autogenerate Token Don't take it from user
-[*] For UpdateUser don't show response
-[*] Check swagger yaml for responses.
-
-*/
+var NewCharacter models.Character
```

Comment:
I think the scope is too big to define as a global variable.
This will most likely cause bugs.
NewUser is same.

Reply:
Okay I will change the scope of the variable.